@ecomplus/storefront-components/src/js/TheProduct.js

```diff
@@ -191,6 +198,10 @@ export default {
       }
     },
 
+    isMarkedAsFavorite () {
+      return ecomPassport.customer.favorites && ecomPassport.customer.favorites.includes(this.body._id, 0)
+    },
+
```

Comment:
```suggestion
    isMarkedAsFavorite () {
      const { favorites } = this.ecomPassport.customer
      return favorites && favorites.includes(this.body._id)
    },

```

usando `this.ecomPassport` e melhor com o `getCustomer`...